.github/workflows/main.yml

```diff
@@ -0,0 +1,55 @@
+name: Build DAPHNE
+
+on:
+  workflow_dispatch:
+  push:
+
+jobs:
+  build:
+    runs-on: ubuntu-22.04
```

Comment:
We should let the automated testing happen on 20.04 imho as we test the newer versions on our dev machines anyway.

Reply:
ok, I can modify this ...